src/main/resources/com/dabsquared/gitlabjenkins/GitLabPushTrigger/config.jelly

```diff
@@ -18,6 +18,9 @@
     <f:entry title="Add note with build status on merge requests" field="addNoteOnMergeRequest">
       <f:checkbox default="true" />
     </f:entry>
+    <f:entry title="Use GitLab CI features (GitLab 8.1 required!)" field="addCiMessage">
```

Comment:
Lucky, I barely know Java and only thought to check for this because of your comment in your other PR :)